extensions/1password/src/generate-password.tsx

```diff
@@ -0,0 +1,11 @@
+import { getPreferenceValues, PreferenceValues } from "@raycast/api";
+import { AuthProvider } from "./v8/components/AuthContext";
+import { RandomPassword } from "./v8/components/RandomPassword";
+
+export default function Command() {
+  return getPreferenceValues<PreferenceValues>().version == "v8" ? (
```

Comment:
```suggestion
  return getPreferenceValues<PreferenceValues>().version === "v8" ? (
```

Let's use strict equal.